blocks/resizer-image-block/index.js

```diff
@@ -138,6 +137,15 @@ export const getResizerParams = (
   // todo: use webp for next gen images spike
   // getParamsByFormat('webp', output);
   getParamsByFormat('jpg', output);
+  // Going to clean the data by stripping off leading slash
+  // and compressing format and quality notation
+  // Since this code is only supporting jpg and filterQuality of 70,
+  // we will just assume as much for now.
+  Object.keys(output).forEach((key) => {
+    if (output[key]) {
```

Comment:
what is this for? the existence check?

Reply:
Defensive coding on the if statement, there could be a case where the key is null.

Reply:
In my mind you can remove this

Reply:
There is a test case where it fails without this check

Reply:
Without that check, it fails in the test "the author api content source block › .transform › should return author with resized params populated"

Reply:
oh right, I see the test case now: ```resized_params: { '158x89': undefined },```

not wildly sure how that could happen, but looks like Matt did it deliberately